app/javascript/components/ReactButtonContainer.jsx

```diff
@@ -15,12 +15,19 @@ class ReactButtonContainer extends Component {
         initStructure: this.props.initStructure,
         audioStreamURL: this.props.audioStreamURL,
         streamDuration: this.props.streamDuration
-      }
+      },
+      structureSaved: true
```

Comment:
Changes to this file fixe a breaking change in react-structural-metadata-editor js package